src/Windows/Containers/Impl/Resources.resx

```diff
@@ -147,6 +147,9 @@
   <data name="Error_ServiceNotAvailable" xml:space="preserve">
     <value>Container service is not available. Check if "{0}" is installed and running.</value>
   </data>
+  <data name="Error_UserNotInDockerUsersGroup" xml:space="preserve">
+    <value>You don't have permissions to use docker. Please add "{0}" to "{1}" group.</value>
```

Comment:
`Docker`. Also, it is hard to localizers to handle constructed sentences since English rules do not always apply. Or at least give loc team hint on what placeholder is for. You may try `Please add user {0} to the group {1}`